.github/workflows/test.yml

```
@@ -15,11 +15,72 @@ jobs:
+        uses: ./
+        with:
+          image: ${{ env.SUCCESS_IMAGE }}
           github-token: ${{ secrets.GITHUB_TOKEN }}
-          image: ghcr.io/joschi/dive:latest
+
+      - name: 'Params: dive-image-registry, dive-image-version'
+        if: always() && steps.pre-setup.outcome == 'success'
+        uses: ./
+        with:
+          image: ${{ env.SUCCESS_IMAGE }}
+          dive-image-registry: ghcr.io/joschi/dive
```

actionlint (1.7.4)

input "dive-image-registry" is not defined in action "Dive Action v2" defined at "./". available inputs are "config-file", "github-token", "image", "image-source"

(action)